services/QuillLMS/app/controllers/api/v1/activities_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Api::V1::ActivitiesController < Api::ApiController
   include QuillAuthentication
 
+  CLASSIFICATION_TO_TOOL = {"connect": "connect", "sentence": "grammar"}
```

Comment:
Nit: our style guide indicates that hash keys should be symbols wherever possible: https://github.com/empirical-org/ruby#symbol-keys